prow/tide/tide.go

```diff
@@ -452,7 +452,7 @@ func (sc *statusController) waitSync() {
 func (sc *statusController) sync(pool []PullRequest) {
 	sc.lastSyncStart = time.Now()
 
-	sinceTime := sc.lastSuccessfulQueryStart.Add(-time.Second)
+	sinceTime := sc.lastSuccessfulQueryStart.Add(-10 * time.Second)
```

Comment:
do we have anything backing this value?

Reply:
What do you mean?

Reply:
do we have any good reason for 10 seconds in particular? :man_shrugging: 
not blocking, just curious.

Reply:
10 seconds of overlap shouldn't affect token usage, but should be enough to ensure that we don't miss any updates.
Its mainly just a guess though. 1 second didn't seem to be enough and 10 > 1. 
/shrug
/hold cancel